recipes/sigmut/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+./sigprofiler -ig GRCh38
+cp * ${PREFIX}/bin/
```

Comment:
It’s usually not a good idea to install Python scripts by just copying them. The normal `pip install` way would at least adjust the "shebang" (the `#!/usr/bin/env python` line) to point to the correct Python version. If that is not done, weird issues can pop up when running the script. You’re also the author of the sigmut package, can’t you add a proper `setup.py` to it so that it will be installable using the normal mechanisms?

Reply:
OK, I will proceed according to your suggestions. Thanks a lot for your kind help @marcelm !

Reply:
This particular `cp` invocation will not copy subdirectories because it is missing a `-r`.

Reply:
Thanks a lot for your kind help @marcelm and @dpryan79 ! I really acknowledge your dedication for solving these issues.